mkdocs.yml

```diff
@@ -136,3 +136,6 @@ nav:
       - vsftpd: apps/vsftpd.md
       - Watchtower: apps/watchtower.md
       - ZNC: apps/znc.md
+ 
+  - Docker Compose Overrides
+      - NGINX Proxy Manger: /Overrides/NginxProxyManager.md
```

Comment:
let's use lowercase lettering the entire url. Also the leading slash is not needed.

Reply:
got it.